.github/workflows/sdk-jobs-pipelines-command_job_in_pipeline-command_job_in_pipeline.yml

```diff
@@ -0,0 +1,52 @@
+name: sdk-jobs-pipelines-command_job_in_pipeline-command_job_in_pipeline
+on:
+  schedule:
+    - cron: "0 */8 * * *"
+  pull_request:
+    branches:
+      - sdk-preview
+    paths:
+      - sdk/**
+      - .github/workflows/sdk-jobs-pipelines-command_job_in_pipeline-command_job_in_pipeline.yml
+      - notebooks/dev-requirements.txt
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - name: check out repo
+      uses: actions/checkout@v2
+    - name: setup python
+      uses: actions/setup-python@v2
+      with: 
+        python-version: "3.8"
+    - name: pip install notebook reqs
+      run: pip install -r notebooks/dev-requirements.txt
+    - name: azure login
+      uses: azure/login@v1
+      with:
+        creds: ${{secrets.AZ_AE_CREDS}}
+    - name: setup SDK
+      run: bash setup.sh
+      working-directory: sdk
+      continue-on-error: true
+    - name: setup CLI
+      run: bash setup.sh
+      working-directory: cli
+      continue-on-error: true
+    - name: run jobs/pipelines/command_job_in_pipeline/command_job_in_pipeline.ipynb
+      run: |
+          sed -i -e "s/<SUBSCRIPTION_ID>/6560575d-fa06-4e7d-95fb-f962e74efd7a/g" command_job_in_pipeline.ipynb
+          sed -i -e "s/<RESOURCE_GROUP>/azureml-examples-rg/g" command_job_in_pipeline.ipynb
+          sed -i -e "s/<AML_WORKSPACE_NAME>/main/g" command_job_in_pipeline.ipynb
+          sed -i -e "s/InteractiveBrowserCredential/AzureCliCredential/g" command_job_in_pipeline.ipynb
+
+          # generate a random workspace name
```

Comment:
These lines are specific to the workspace.ipynb notebook, and we should remove them before we check these files in. The others are generic across notebooks.

Reply:
Removed unwanted lines